config/rapidez/frontend.php

```diff
@@ -21,6 +22,9 @@
         // 'material',
     ],
 
+    // The attribute of the color swatch, used in the active filtering
+    'color_attribute' => 'color',
```

Comment:
How does this work? From Magento you can have multiple swatch attributes with their own (color) values.

Reply:
See my comment here; https://github.com/rapidez/core/pull/587/files#diff-e28427a88f3021679abe0aa8a08b6796959c85a443f2b127192669829c0a663aR41